content/doc/book/system-administration/admin-password-reset-instructions.adoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: section
+---
+
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= Reset Jenkins administrator password
+
+As a system administrator, you can reset the Jenkins admin password.
+In case you have forgotten or lost the password, this page provides instructions on how you can reset the admin password.
+
+== Reset the administrator password
+
+1. Log in to your Jenkins controller and edit the Jenkins configuration file (`config.xml`) inside your `jenkins/` or `$JENKINS_HOME` folder.
+2. Look for the **useSecurity** option and change it from **true** to **false** manually.
+3. Save your file and close it.
+4. Restart the Jenkins service to apply changes. (This time it will not ask for a password.)
+5. On the dashboard, click on **Manage Jenkins** in the navigation pane on the left side of the page.
+6. In **Manage Jenkins**, in the **Security** section you will get **Configure Global Security**. Click on it.
+7. In **Security Realm**, click on the checkbox for **Jenkins' own user database** and save this. You will get redirected to the Manage Jenkins page afterwards.
+8. In **Security**, click on **Manage Users**.
+9. You will see a screen showing your username with multiple tabs. Click on the **Settings** icon.
+10. Search for the **Password** section to change your password.
+
+After changing the password, you will be able to log into your Jenkins instance again using the same username but the new password that you have just set.
```

Comment:
```suggestion
After changing the password, you will be able to log into your Jenkins instance again using the same username but the new password that you have just set.
```
There are no instructions as to how to re-apply the security and lock the controller down, which is a key element that is missing. Without this part of the instruction, this leaves users vulnerable to anyone accessing their controller. The steps to lock the controller back down are necessary, and this cannot be merged without including that. If they are not included as steps, this needs to be made clear and why it is important in a separate paragraph/section.

Reply:
I agree with this. These are necessary things to be mentioned.